main.go

```diff
@@ -170,6 +170,7 @@ func retryFailedTests(testRunner runner.Rspec, maxRetries int) int {
 // logErrorAndExit logs an error message and exits with the given exit code.
 func logErrorAndExit(exitCode int, format string, v ...any) {
 	fmt.Printf(format, v...)
+	fmt.Print("\n")
```

Comment:
Drive-by suggestion:

1. A shorter way to write this is `fmt.Println()`
2. Alternatively you could use `format+"\n"` in the line above

Reply:
We love driveby suggestions, thanks @DrJosh9000! @nprizal want to make the change and ship?